src/logger.js

```diff
@@ -1,14 +1,48 @@
+import chalk from 'chalk'
+
 const LOG_LEVELS = {
   debug: 1,
-  info: 2,
-  silent: 3
+  protractor: 2,
+  info: 3,
+  silent: 4
 }
 
-export default function (levelName, message) {
-  let currentLevel = LOG_LEVELS[process.env.PROTRACTOR_FLAKE_LOG_LEVEL] || LOG_LEVELS.info
-  let incomingLevel = LOG_LEVELS[levelName]
+const DEFAULT_COLOR = 'magenta'
 
-  if (incomingLevel >= currentLevel) {
-    process.stdout.write(message)
+class Logger {
+  constructor (color) {
+    this.color = null
+    if (chalk.supportsColor) {
```

Comment:
Since we are doing a `chalk.supportsColor` check in the `colorize` method I think we can leave this outer `if` out